app/assets/javascripts/discourse/app/models/login-method.js

```diff
@@ -13,6 +13,11 @@ const LoginMethod = EmberObject.extend({
     return this.title_override || I18n.t(`login.${this.name}.title`);
   },
 
+  @discourseComputed
+  a11yTitle() {
```

Comment:
maybe we go with `screenReaderTitle` now ?

Reply:
Yes sorry missed that; I think I replaced `a11y_` in this file and not just `a11y`